app/models/package.rb

```diff
@@ -60,6 +60,7 @@ class Package < ApplicationRecord
   end
 
   validates :package_type_id, presence: true
+  validates :value_hk_dollar, presence: true, if: -> { !new_record? && !box_or_pallet? }
```

Comment:
why do we need to have a condition on `new_record`?

Reply:
Here we already have a `before_create` callback to set `value_hk_dollar` when its blank.
https://github.com/crossroads/api.goodcity/blob/master/app/models/concerns/valuation_calculator.rb#L8-L12 

Hence skipping this validation for new-records.

Reply:
thanks, actually `before_validation` makes a lot more sense here... if we are computing a value, we really should check it meets our validation requirements.